src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const payment = 40;
+
+  if (days >= 7) {
+    return days * payment - 50;
+  }
+
+  if (days >= 3 && days <= 7) {
+    return days * payment - 20;
+  }
+
+  if (days <= 2) {
+    return days * payment;
+  }
 }
 
+calculateRentalCost();
```

Comment:
The function `calculateRentalCost` is called without any arguments. This will result in an error because the function expects a `days` parameter. You should provide a valid number of days when calling this function.